### ISSUE TITLE
Add Dashboard playwright tests to ensure UI functionality and integration with app host

### ISSUE THREAD

**Issue by joperezr** [dotnet/aspire#4295]
As discussed offline, we want to add playwright tests to the dashboard in order to have coverage across different layers. The ones we mentioned we'd want to cover are:
- Mocking Data and looking at UI elements.
- Running Dashboard from the AppHost and doing some tests. This will help validate the protocol of communication between dashboard and apphost. #3737
- End-to-End tests using the workload.

I'm temporarily assigning this to @kvenkatrajan , but please feel free to re-assign as appropriate. (I believe you mentioned @adamint was already looking into some of this?)

**Comment by kvenkatrajan** [comment]
@adamint  can you link the Playwright PR that you are working on to this please?

**Comment by adamint** [comment]
#4386

**Comment by joperezr** [comment]
Depending on resource availability, we may need to move this off from 8.1

**Comment by davidfowl** [comment]
@radical we have playwright tests now right?

**Comment by radical** [comment]
> @radical we have playwright tests now right?

Yes, we have some in `Aspire.Dashboard.Tests`, and some in the workload tests. The ones mentioned here need to be added.
